WinFormsUI/MainForm.cs

```diff
@@ -62,8 +65,10 @@ private void MainForm_Load(object sender, EventArgs e)
 		private void LoadTransactions()
 		{
 			var currentDirecory = Directory.GetCurrentDirectory();
-			var files = Directory.GetFiles(currentDirecory + "/pb").Select(f => ("pb", (Stream)File.OpenRead(f)));
-			StateManager.LoadTransactions(files);
+			//var files1 = Directory.GetFiles(currentDirecory + "/usb").Select(f => ("usb", (Stream)File.OpenRead(f)));
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=5924375)